app/views/layouts/application.html.slim

```diff
@@ -42,7 +42,7 @@ html lang="#{I18n.locale}"
     .site-wrap
       = render 'shared/i18n_mode' if FeatureManagement.enable_i18n_mode?
       = render 'shared/usa_banner'
-      - if signed_in?
+      - if user_fully_authenticated?
```

Comment:
I did! The white header is only supposed to be shown to users who have completed the sign in process.

You can check out the invision screens I linked in the issue.